compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/awt/ComposePanelTest.kt

```diff
@@ -360,4 +366,46 @@ class ComposePanelTest {
             }
         }
     }
+
+    @Test
```

Comment:
Perhaps explain what this tests, and add a link to the github issue.

Reply:
Done